influence_benchmark/RL/KTO_training.py

```diff
@@ -71,8 +80,10 @@ def format_dataset(example):
         model.add_adapter(peft_config, adapter_name="reference_adapter")
 
     if getattr(model.config, "pad_token_id", None) is None:
-        tokenizer.pad_token = tokenizer.eos_token
-        model.config.pad_token_id = tokenizer.eos_token_id
+        pad = "<|reserved_special_token_198|>"  # Llama doesn't have a pad token, so we use a reserved token
```

Comment:
this is only for llama 3 not 3.1, I thought I changed this but maybe it was somewhere else or in a later commit